frontend/src/metabase/dashboard/components/DashCard/DashCardTabMenu/DashCardTabMenu.tsx

```diff
@@ -39,8 +46,10 @@ export function DashCardTabMenu({ dashCardId }: DashCardTabMenuProps) {
           },
         }),
       );
+
+      trackCardMoved(checkNotNull(dashboardId));
```

Comment:
Please move tracking to the redux action `moveDashCardToTab`

Reply:
The action right now is just a plain action (no thunk)
There was a suggestion about converting it to a thunk to also include the undo toast, but it was reverted following the same discussion as there wasn't consensus about it https://github.com/metabase/metabase/pull/34961#discussion_r1371280900

Should I re-convert it to a thunk that includes the analytics event?